### DISCUSSION TITLE
Feedback for the demo test

### DISCUSSION THREAD

**Discussion by ilducesaint** [akatori-game/feedback#115] [Feedback]
Hey folks,
this is my feedback after playing the game, i hope it helps, and sorry if my english is sloppy, it's not my first language, cheers and good luck with the game!!.
1.-first attack no delay for evade, intended game design?
2.-second,third attack a little very little delay for evade, intended?? - make me use more the first attack than the others cuz is faster and let's me evade easier, i mostly spammed first attack, to be more safe while attacking and engaging.
3.-mid air down-attack but not up-attack, there is in-ground up-attack tho, intended game design?
4.-maybe add charge attack like charge beam of samus, like smash attacks, more time charging more powerful
5.-no double jump, maybe later in the full release as upgrade?
6.-no parry or shield, intended game design? i love parry's but that my personal opinion
7.-ground infinite evades, intended game design?, well i like infinite evades honestly 
8.-no crouch, intended game design?
9.-enemies dont respawn using rest, only when dying or changing scenes (map loading dont know the game design term) - intended game design?
10,-if you attack then evade only evades the direction you are facing you cant evade backwards, i mean when you use the analog stick the opposite direction while evading - intended game design?
11.-(using analog stick) very tight control for go down a platform, if you slightly diagonal down you cant go down the platform (seems to only affect fast pacing traversal, when you want to be fast in where you are going/where you want to go)
12.-there is a spot where you can evade completely the 1st boss ball attack, where the path is closed by a rock wall, there is a little spot where you can stand (in the very right place of the "arena boss".
Thank you for reading, have a nice day!

**Comment by XenoWarrior** [comment]
Hey @ilducesaint, thanks for the time and effort!

I see some of these are bugs.
If it's not too much to ask, could you please put obvious bugs into their own bug tickets at:
https://github.com/akatori-game/feedback/issues/new?assignees=&labels=&template=bug_report.md&title=

Thank you!

**Reply by ilducesaint** [reply]
hello @XenoWarrior, thanks for your reply!

can you tell me which one's are bugs by telling me with the numbers, which ones are the bug ones?.
Since i didn't make the game, i can't identify what are the intended features, and what doesn't suppose to happen.
You can help me a lot by telling me which some of these are bugs, so i can pass them to the bug tickets and i don't screw up by posting
wrongly a "not a bug" in the bug tickets.
Thank you and have a nice day!
